schema/deploy/warehouse/site/data.sql

```diff
@@ -5,6 +5,8 @@ begin;
 
 insert into warehouse.site(identifier, details)
     values
+        ('AIRSSwabNSend',                           '{"category": "community",  "type": "swab-n-send",
```

Comment:
Double checking here, is this the swab site being used by the lab for these samples? I spot checked an airs sample in the lims and it had sample_origin `airs` but I didnt see a swab site listed.

Reply:
I thought this was only used on the ID3C side--if it's coming from the lab without a swab site included then I can use Dave's suggestion and remove the swab site logic from the ETL entirely.